src/ledger/LedgerManagerImpl.cpp

```diff
@@ -604,6 +612,10 @@ bool
 LedgerManagerImpl::hasLastClosedSorobanNetworkConfig() const
 {
     releaseAssert(threadIsMain());
+    if (!mLastClosedLedgerState)
```

Comment:
A comment about racing on startup here would be useful, as it seems on first glance it looks like this should just throw.